Userland/Libraries/LibWeb/CSS/Properties.json

```diff
@@ -30,6 +30,103 @@
       "align-self"
     ]
   },
+  "animation": {
+    "affects-layout": true,
```

Comment:
(hmm… this depends on what exactly is animated… 🤔)

Reply:
Not exactly, changing the animation property itself will never affect layout, the animation _running_ can of course, affect layout by changing properties that do.

Reply:
hah! sounds reasonable!